src/tools/fuzzing.h

```diff
@@ -406,20 +406,14 @@ class TranslateToFuzzReader {
     Index num = upTo(3);
     for (size_t i = 0; i < num; i++) {
       // Events should have void return type and at least one param type
-      Type type = pick(i32, i64, f32, f64);
-      std::string sig = std::string("v") + getSig(type);
       std::vector<Type> params;
-      params.push_back(type);
       Index numValues = upToSquared(MAX_PARAMS - 1);
-      for (Index i = 0; i < numValues; i++) {
-        type = pick(i32, i64, f32, f64);
-        sig += getSig(type);
-        params.push_back(type);
+      for (Index i = 0; i < numValues + 1; i++) {
+        params.push_back(pick(i32, i64, f32, f64));
       }
       auto* event = builder.makeEvent(std::string("event$") + std::to_string(i),
                                       WASM_EVENT_ATTRIBUTE_EXCEPTION,
-                                      ensureFunctionType(sig, &wasm)->name,
-                                      std::move(params));
+                                      Type(params));
```

Comment:
as `MAX_PARAMS` is more than 1, this may create multivalue types... are we ready for that?

Reply:
yes, it looks like @aheejin's binary parsing code is able to generate multiple pops. The fuzzer could generate multivalue events before this change, too.